test/utils.py

```diff
@@ -55,6 +55,10 @@ def LONG_TESTS():
     # override LONG_TESTS with environment variable if available
     if os.getenv("LONG_TESTS"):
         LONG_TESTS = int(os.getenv("LONG_TESTS"))
+        if LONG_TESTS > 0:
+            LONG_TESTS = True
+        else:
+            LONG_TESTS = False
```

Comment:
This implementation will still return 0 (not bool) if there's no environment variable.

I think this function can be greatly simplified if we assume that LONG_TESTS can only be 0 or 1:
```py
def LONG_TESTS() -> bool:
    env_var = os.getenv("LONG_TESTS")
    if env_var:
        return bool(int(env_var))
    return False
```

Reply:
I think we probably should not do it because the cli returns errors if you write like LONG_TESTS == 0 and not for LONG_TESTS>0.

Reply:
However I will implement something along those lines